jekyll/_cci2/release/releases-overview.adoc

```diff
@@ -51,13 +51,124 @@ Configure a release environment to install the CircleCI release agent into your
 
 Developers can monitor releases in the CircleCI web app, and run commands to manage those releases from the UI. Commands are monitored by CircleCI and relayed to the release agent, which then runs the commands for you.
 
+[#releases-concepts]
+== Concepts
+
+The following sections describe concepts that will help you fully understand how CircleCI releases integration and capabilities.
```

Comment:
This sentence seems off. Maybe it should be `fully understand CircleCI Releases integration and capabilities.`